corehq/apps/sms/templates/sms/usage_report.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'hqwebapp/base_section.html' %}
```

Comment:
Since this page runs an icds-specific command, I think the template belongs in`custom/icds`.

Reply:
Done